oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -230,7 +243,7 @@ private boolean indexProperty(String path,
 
             if (pd.fulltextEnabled() && includeTypeForFullText) {
                 for (String value : property.getValue(Type.STRINGS)) {
-
+                    logLargeStringProperties(property.getName(), value);
```

Comment:
I wanted to add this comment at line 241 but apparently I can't do that.

I think we should split `dirty |= addTypedFields(doc, property, pname, pd);` and see if `addTypedFields` returns true. If so, log here for large property indexing after checking that tag was `STRING`.

Reply:
Answered above.